client/src/pages/LogIn_SignUp.js

```diff
@@ -0,0 +1,208 @@
+import {
+  TextField,
+  Button,
+  Typography,
+  Link,
+  Paper,
+  Grid,
+  Box,
+  makeStyles,
+} from '@material-ui/core';
+import React, { useEffect, useState, useCallback } from 'react';
+import { useLocation } from 'react-router-dom';
+
+const useStyles = makeStyles({
+  textField: {
+    width: '50%',
+  },
+  bold: {
+    color: 'black',
+  },
+  seperator: {
+    width: '10%',
+    border: '1px solid #00e676',
+  },
+  formPadding: {
+    padding: '20px',
+  },
+});
+
+const LogIn_SignUp = () => {
+  //state declaration
+  const [signInEmail, setEmail] = useState('');
+  const [signInPassword, setPassword] = useState('');
+  const [registerName, setName] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+  const [lengthValidation, setLengthValidation] = useState(true);
+  const [confirmValidation, setConfirmValidation] = useState(true);
+
+  const classes = useStyles();
+
+  //Determines which form elements to render based on the path
+  const signInForm = useLocation().pathname === '/login' ? true : false;
```

Comment:
You don't need a ternary operator here if you're only returning true or false
```suggestion
  const signInForm = useLocation().pathname === '/login';
```

Reply:
Removed the check all together since I have separated the forms into their own pages